sdk/python/kfp/components/placeholders.py

```diff
@@ -146,6 +146,16 @@ def to_placeholder_string(self) -> str:
         return self._TO_PLACEHOLDER.format(**self.to_dict())
 
 
+class ExecutorInputPlaceholder(base_model.BaseModel,
+                               RegexPlaceholderSerializationMixin):
+    """Class that represents executor input placeholder."""
+    _TO_PLACEHOLDER = "{{$}}"
```

Comment:
nit: prefer single quotes when no double quote character is contained inside the string

(throughout)

Reply:
Done